modules/getting-started/proc-customize-rhdh-homepage.adoc

```diff
@@ -6,7 +6,23 @@ To access the Home page in {product}, the base URL must include the `/developer-
 * JSON files hosted on GitHub or GitLab.
 * A dedicated service that provides the Home page data in JSON format using an API.
 
-== Using hosted JSON files to provide data to the Learning Paths
+== Using hosted JSON files to provide data to the Home page
+
+When using the `app-config`, you can do the following:
+
+
+* Customize and extend the default Home page layout with additional cards that appear based on the plugins you have installed and enabled. 
+* Change quick access links.  
+* Add, reorganize, and remove the following available cards:
++
+** Search bar
+** Quick access
+** Headline
+** Markdown
+** Placeholder
+** Catalog starred entities
```

Comment:
As discussed in Google meet, I think this is a good intro into the section 4 at all.